components/eam/src/physics/cam/nucleate_ice.F90

```diff
@@ -141,11 +141,7 @@ subroutine nucleati(  &
    real(r8) :: nidep                     ! nucleated number from deposition nucleation
    real(r8) :: nimey                     ! nucleated number from deposition nucleation (meyers)
    real(r8) :: n1, ni                    ! nucleated number
-   real(r8) :: tc, A, B, regm            ! work variable
-   real(r8) :: esl, esi, deles           ! work variable
-
-   
-   real(r8)  :: nimeystp                                  ! nucleated number from ice nucleation (meyers) at STP  
+   real(r8) :: tc, A, B, regm            ! work variable  
```

Comment:
maybe also replace A and B with A_coef and B_coef?

Reply:
@tangshuaiqi Thanks for the suggestion. I'm trying to do one PR for one subroutine at a time, although I changed the .lt./.ge./.le./ to </>=/<= and removed some variables in subroutine nucleati that are no longer needed. The next step is to refactor subroutine nucleati where you suggested replacing A and B. Maybe I need to make a new subroutine for the calculation using A and B. There are a few things needed for subroutine nucleati. 

@singhbalwinder what would you suggest? I can also refactor nucleati in this PR.

Reply:
Okay... I did not notice that this is in another subroutine.

Reply:
Since this is ready, let's merge it and start with the new PR for the next subroutine. It is easier to see changes in small chunks.